services/cache/lru.go

```diff
@@ -186,47 +185,58 @@ func (l *LRU) set(key string, value interface{}, ttl time.Duration) error {
 }
 
 func (l *LRU) get(key string, value interface{}) error {
-	if ent, ok := l.items[key]; ok {
-		e := ent.Value.(*entry)
 
```

Comment:
There's an empty line here. We can remove it.

Reply:
@streamer45 do you want me to push an update to this PR removing empty line?

Reply:
Yes please :)